sw/device/silicon_creator/lib/drivers/pinmux.h

```diff
@@ -23,6 +23,13 @@ extern "C" {
  */
 void pinmux_init(void);
 
+/**
+ * Read the SW_STRAP value considering weak/strong pull resistors.
+ *
+ * @return The strapping value 0-63.
+ */
+uint32_t pinmux_read_straps(void);
```

Comment:
Should this function document how the value is encoded (ie 2 bits per pin with weak/strong)?